assets/blocks/BlackAndWhite.block

```diff
@@ -0,0 +1,56 @@
+{
+    "displayName": "Black and White",
+    "categories" : ["BlackAndWhite", "Black", "White"],
+    "hardness": 2,
+    "tiles" : {
+        "topBottom" : "BlackAndWhiteTop",
+        "sides" : "BlackAndWhiteSide"
+    },
+
+    "family": "BlackAndWhite",
+
+    "lone_block": {
+	    "tiles" : {
+	        "topBottom" : "BlackAndWhiteTop",
+	        "sides" : "BlackAndWhiteSide"
+	    },
+    	"shape": "engine:cube"
+    },
+
+<<<<<<< HEAD
```

Comment:
This seems like a failed merge attempt. No use reviewing the PR, till you fix this.

I highly recommend you locally test your files before pushing them.